components/composer/src/Composer.svelte

```diff
@@ -74,7 +74,6 @@
   export let access_token: string = "";
 
   export let value: Message | void;
-  export let message_with_body: Message | void;
```

Comment:
[semver](https://semver.org/) dictates that deprecating any functionality requires a minor release and the actual functionality shouldn't be removed until we release a new major version since this was introduced in `@nylas/components-composer@1.1.6-canary.7`. In the case that an end user may have installed any version from `1.1.6-canary.7` to `1.1.6-canary.10` we need to only deprecate this functionality and remove it completely in the next major version. We can just make note of it by adding a comment (eg. `// TODO: Deprecated. Remove in v2.0.0`) 

![image](https://user-images.githubusercontent.com/34139730/154285828-af50a524-e06f-4f3e-830f-3925162949fe.png)